exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook_requirements.txt

```diff
@@ -8,3 +8,4 @@ sqlalchemy_exasol==4.6.3
 stopwatch.py==2.0.1
 boto3==1.26.163
 exasol-notebook-connector @ git+https://github.com/exasol/notebook-connector.git@main
+pickleshare==0.7.5 # Needed because of https://github.com/exasol/ai-lab/issues/289. Possibly can be removed in the future.
```

Comment:
```suggestion
pickleshare==0.7.5 # Needed because of https://github.com/exasol/ai-lab/issues/289. Potentially can be removed as soon as an update for ipy available.
```

Reply:
Is this the dependency to `ipywidgets` in file `ansible/roles/jupyter/files/jupyter_requirements.txt` then?

Reply:
If true, then there pypi offers [ipywidgets 8.1.3](https://pypi.org/project/ipywidgets/8.1.3/)  which is later than 8.1.1 currently used by the ai-lab.

Reply:
Fixed according to our discussion on the phone.